src/TouristInfoCards.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div class="card" style={{ width: "18rem" }}>
```

Comment:
Remember that when writing a React application you have to convert `class` to `className`!